sympy/logic/boolalg.py

```diff
@@ -186,6 +197,10 @@ class BooleanFalse(with_metaclass(Singleton, BooleanAtom)):
     False they act bitwise on 0. Functions in the logic module will return this
     class when they evaluate to false.
 
+    Notes
+    ======
+    See note in :py:class`sympy.logic.boolalg.BooleanTrue`
```

Comment:
The text still does not explain what kind of note can be found at the other end of the link.